imap/message.c

```diff
@@ -693,7 +694,11 @@ int imap_read_headers(struct ImapData *idata, unsigned int msn_begin, unsigned i
     mutt_progress_init(&progress, _("Evaluating cache..."), MUTT_PROGRESS_MSG,
                        ReadInc, msn_end);
 
-    snprintf(buf, sizeof(buf), "UID FETCH 1:%u (UID FLAGS)", uidnext - 1);
+    if (snprintf(buf, sizeof(buf), "UID FETCH 1:%u (UID FLAGS)", uidnext - 1) < 0)
```

Comment:
Is there any reasonable situation in which this snprintf could fail, that you can think of?

Reply:
That's a good point, I'll revert this one.